container-storage-setup.sh

```diff
@@ -755,26 +756,33 @@ create_partition() {
     Fatal "udevadm settle after partition creation failed. Exiting."
   fi
 
-  if ! wait_for_dev ${dev}1; then
-    Fatal "Partition device ${dev}1 is not available"
+  part=$(query_first_child $dev)
+
+  if ! wait_for_dev ${part}; then
+    Fatal "Partition device ${part} is not available"
   fi
+  echo $part
+}
+
+query_first_child() {
+  lsblk -npl -o NAME "$1" | tail -n +2 | head -1
```

Comment:
ok, we can leave it as it is. I think I was much more familiar with head -2. For tail -n +2, I had to read the man page and that's one reason I thought it was simpler. 

No big deal. I will test it.